pyproject.toml

```diff
@@ -14,10 +14,15 @@ smqtk-detection = {git = "https://github.com/Kitware/SMQTK-Detection", rev = "ma
 torch = "^1.10.2"
 torchvision = "^0.11.3"
 trimesh = {extras = ["all"], version = "^3.10.2"}
+pytorchvideo = "^0.1.5"
 
 [tool.poetry.dev-dependencies]
 ipython = "^8.0.1"
 
 [build-system]
 requires = ["poetry-core==1.0.7"]
```

Comment:
I just found out that we can update this to `==1.0.8`, which supports installing the package as editable via pip (supports PEP660 now). This is needed to install editable via pip as in the Dockerfile comment.